cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${projectdir} build.vars)
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+
+helm_tag="$(cat ${projectdir}/_output/version)"
```

Comment:
Looks like we're depending on the `_output/version` already having been generated; do we already guarantee that this file exists before we get to this line? I checked the Makefile but I wasn't able to find a line that does that.

Reply:
`_output` folder is generated at `build` target. Since `test-integration` is executed after `build`, this folder should already exist.